### DISCUSSION TITLE
Web Cryptography API interoperability layer

### DISCUSSION THREAD

**Discussion by panva** [panva/jose#130] [Q&A]
Anyone using [PeculiarVentures/webcrypto-liner](https://github.com/PeculiarVentures/webcrypto-liner)? What are your experiences? Can one use it to get `jose` in IE11 to work? Please let others know your experiences here.

@PeculiarVentures @rmhrisk what are the individual applied fixes to interoperability when requiring just webcrypto-liner without any of the js implementation crypto runtimes for each browser vendor/version? I recall IE11 implements an older version of the Web Cryptography API specification that didn't rely on Promises, does webcrypto-liner fix those? What about the `jwk` format key exports that resulted in an JSON serialized in an ArrayBuffer instead of an object? What about recently fixed [chromium issues](https://bugs.chromium.org/p/chromium/issues/detail?id=1135489)?

**Comment by rmhrisk** [comment]
Ie11 and liner work fine together. You can see what works with liner here: https://peculiarventures.github.io/pv-webcrypto-tests/
